packages/react/src/analytics/integrations/AnalyticsApi/types/types.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  type EventData,
+  type EventTypes,
+  type IntegrationOptions,
+  type LoadIntegrationEventData,
+  type PageTypes,
+  type SetUserEventData,
+  type TrackTypesValues,
+} from '@farfetch/blackout-analytics';
+
+export interface AnalyticsApiIntegrationOptions extends IntegrationOptions {
+  conversionsAPI: {
+    eventsList: Array<`${EventTypes}` | `${PageTypes}`>;
+    testEventCode?: string;
+  };
+}
+
+export type AnalyticsApiTrackData = {
+  data:
```

Comment:
From the implementation code, I believe only `EventData<TrackTypesValues>` will be possible to be sent. Should this integration handle `onSetUser` and `loadIntegration` events as well? If so, then the implementation must be changed to handle that case (track method will not be called by analytics in those cases).

Reply:
For this point, we have discussed this already and at the moment server side don't require these events. Of course, we could implement as well but it will be increased requests that won't be processed by server.